### ISSUE TITLE
[EPCO-16] Neural network can be trained on sim data

### ISSUE THREAD

**Issue by hatfield-c** [hatfield-c/epco#16]
A normalized ReLU neural network will need to be instantiated in Python using the PyTorch library.

Using the data loader from EPCO-#8, this implementation will focus on the training infrastructure

**Comment by hatfield-c** [comment]
This issue is now complete, and will thus be closed.
